cfme/tests/automate/test_domain.py

```diff
@@ -393,3 +397,59 @@ def test_copy_to_domain(domain):
     method = domain.browser.create_view(MethodCopyView)
     method.flash.wait_displayed()
     method.flash.assert_message("Copy selected Automate Method was saved")
+
+
+@pytest.fixture(scope="function")
+def new_user(request, appliance):
+    """This fixture creates custom user with tenant attached"""
+    tenant = _tenants(request, appliance)
+    role = appliance.rest_api.collections.roles.get(name="EvmRole-super_administrator")
+    group = _groups(request, appliance, role, tenant=tenant)
+    user, user_data = _users(request, appliance, group=group.description)
+    yield appliance.collections.users.instantiate(
+        name=user[0].name,
+        credential=Credential(principal=user_data[0]["userid"], secret=user_data[0]["password"]),
+    ), tenant
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1678122])
+@pytest.mark.ignore_stream("5.10")
+def test_tenant_attached_with_domain(request, new_user, domain):
+    """
+    Note: This RFE which has introduced extra column for tenant on domain all view
+
+    Bugzilla:
+        1678122
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        startsin: 5.11
+        casecomponent: Automate
+        setup: Create new user
+        testSteps:
+            1. Log in with admin. Create automate domain and navigate to domains all page
+            2. Log in with new user. Create automate domain and navigate to domains all page
+        expectedResults:
+            1. Automate domain should be assigned with tenant - 'My Company'
+            2. Automate domain should be assigned with new user's tenant
+    """
+    # Domain created by admin user attached with root tenant "My Company"
+    user, tenant = new_user
+    view = navigate_to(domain.parent, "All")
+    assert view.domains.row(name=domain.name)["Tenant"].text == "My Company"
+
+    # Log in with new user
+    with user:
+        # Domain created by new user attached with new tenant assigned to this user
+        new_domain = domain.appliance.collections.domains.create(
```

Comment:
**Optional** : You could use `domain.parent.create` instead of `domain.appliance.collections.domains.create`.